src/initialization/logs.py

```diff
@@ -0,0 +1,48 @@
+# Log Parser for RTI Connext.
```

Comment:
I would say to add these logs into the config module. They are *similar* and at the moment there isn't enough logs to split into a new module.